src/app/static/src/tests/components/plots/choroplethFilter.test.ts

```diff
@@ -476,14 +476,25 @@ describe("ChoroplethFilters component", () => {
             .toStrictEqual([FilterType.Quarter, "1"]);
     });
 
-    it("shows quarter filter if data type is output", () => {
```

Comment:
didn't need these tests any more as now we're testing that all the filters are there in the right order